l10n_es_ticketbai/models/account_invoice.py

```diff
@@ -419,15 +416,34 @@ def tbai_is_invoice_refund(self):
             res = False
         return res
 
-    def split_invoice_number(self):
-        m = re.match(INVOICE_NUMBER_RE, self.number)
-        return m.group(1), m.group(2)
+    def _get_invoice_sequence(self):
+        journal_id = self.journal_id
+        if self.env['ir.module.module'].search([
```

Comment:
Igual con un hasattr(journal_id, "invoice_sequence_id") queda más limpio.

Reply:
Me parece correcto ajustar el IF, ¿quizás mejor `if invoice_sequence_id in journal_id:`? ¿Más limpio?